db/migrate/20191128111048_add_missing_foreign_keys.rb

```diff
@@ -0,0 +1,13 @@
+class AddMissingForeignKeys < ActiveRecord::Migration[5.2]
+  def change
+    add_foreign_key :api_secrets, :users, on_delete: :cascade
+    add_foreign_key :classified_listings, :users, on_delete: :cascade
+    add_foreign_key :identities, :users, on_delete: :cascade
+    add_foreign_key :notification_subscriptions, :users, on_delete: :cascade
+    add_foreign_key :page_views, :articles, on_delete: :nullify
```

Comment:
`:nullify` means "set the `article_id` of the page view when the article is deleted"

Reply:
I don't see any reason to keep these since we are expunging all of the other user data so I am in favor of removing them. I also don't see a mass exodus of users happening so I think the small downtick in an article page view is not going to be a big deal. @benhalpern, thoughts?